### ISSUE TITLE
Upload failed when doccano is deployed with 2 replica in kubernetes

### ISSUE THREAD

**Issue by YosuaMichael** [doccano/doccano#1475]
Hi, I deployed doccano in kubernetes with 2 replica, connected to same external Postgresql and Rabbitmq.
Then when I tried to upload a dataset I got the following error message:
```
[2021-08-20 09:02:54,137: INFO/MainProcess] celery@doccano-demo-7c94b9d45-4z6v9 ready.
[2021-08-20 10:53:21,827: INFO/MainProcess] Task api.tasks.export_dataset[2300c812-281d-4258-802a-d0bcc5bc1513] received
Error moving temporary file to permanent storage location
[2021-08-20 10:56:52,627: INFO/MainProcess] Task api.tasks.injest_data[004c53b6-fe82-45fc-b4db-7ea16691db25] received
[2021-08-20 10:56:52,659: ERROR/ForkPoolWorker-2] Task api.tasks.injest_data[004c53b6-fe82-45fc-b4db-7ea16691db25] raised unexpected: FileNotFoundError(2, 'No such file or directory')
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/celery/app/trace.py", line 450, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/celery/app/trace.py", line 731, in __protected_call__
    return self.run(*args, **kwargs)
  File "/doccano/backend/api/tasks.py", line 111, in injest_data
    example = next(it)
  File "/doccano/backend/api/views/upload/dataset.py", line 68, in __iter__
    yield from self.load(filename)
  File "/doccano/backend/api/views/upload/dataset.py", line 184, in load
    with open(filename, encoding=encoding) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/doccano/backend/media/5ohBU5hebFUcSo7Qi8Z4gq/YfTPmYLuJCzdFCR62KfVCM/sample_poi.ndjson'
```

If I scale down to only 1 replica, then I can upload successfully.
Is there any suggestion on how to scale doccano to more than 1 replica?

Your Environment
---------
*   Operating System: According to dockerfile python:3.8.6-slim-buster
*   Python Version Used: 3.8.6
*   When you install doccano: 14 August 2021
*   How did you install doccano (Heroku button etc): in AKS (kubernetes cluster in Azure)

**Comment by YosuaMichael** [comment]
Hi, I have been debugging the problem and I think I found the reason why (although I am still not sure the best way to fix it).

So to illustrate the problem, consider the scenario where we deploy doccano into two server A and B. I have two scenarios that produce different errors:
Scenario 1:

- When we are in upload page and choose file (or drop them) it will be successful and a temporary file (in `/doccano/backend/filepond-temp-uploads` folder) is created on A but note that B **dont** have the temporary file!
- After that when we click the button Injest, the load balancer actually direct this request to B. And since B dont have the temporary file, it got error when it tried to move it into the folder `/doccano/backend/media`
- This will result in error message: `Error moving temporary file to permanent storage location`

Scenario 2:

- When we are in upload page and choose file (or drop them) it will be successful and a temporary file (in `/doccano/backend/filepond-temp-uploads` folder) is created on A but note that B **dont** have the temporary file!
- After that when we click the button Injest, it seems loading and the load balancer direct the request to A (so A have both file in `filepond-temp-uploads` and `media` folder). However after a while the injest still failed and got error message from server B.
- The error message in B is like: `FileNotFoundError: [Errno 2] No such file or directory: '/doccano/backend/media/5ohBU5hebFUcSo7Qi8Z4gq/YfTPmYLuJCzdFCR62KfVCM/sample_poi.ndjson'`

So basically for scenario 1, the error is clear because the temporary file is not shared and the load balancer assign different server when we drop the file and when we click injest.

For scenario 2, I am not too sure why, but my guess is after the file properly uploaded, a task is created (through rabbitmq) to read and process it into DB. And the one doing this task is the server that dont have the file.

I hope this make the problem clearer and I hope this can be fixed... (feel free to ask for more detail!).

Thanks,
Yosua
